sql-plugin/src/main/java/com/nvidia/spark/rapids/GpuColumnVector.java

```diff
@@ -455,7 +455,7 @@ public void releaseReferences() {
     }
   }
 
-  private static DType toRapidsOrNull(DataType type) {
+  private static DType atomicAndNullTypeToRapidsOrNull(DataType type) {
```

Comment:
The original was better given `getNonNestedRapidsType` still exists and is a wrapper around this new name.